Makefile.am

```diff
@@ -31,6 +33,7 @@ GEN_UNIFIED_CPP=$(TOOLSDIR)/gen-unified-sources.py
 # The first 7 are standard automake variables
 CLEANFILES =                   # Files to remove on clean
 bin_PROGRAMS =                 # Binaries built
+lib_LTLIBRARIES =              # Libtool convenience libraries
```

Comment:
We should probably actually just get rid of the number here and say something like "these". We have enough maintenance burdens as it is. =)